legacy_app.py

```
@@ -2155,22 +2176,83 @@ async def bmi_endpoint(req: BMIRequest) -> Dict[str, Any]:
+    # 3) Preserve legacy /plan category behavior
+    # RU: minors должны получать строковую категорию, даже если engine.category=None.
+    # EN: minors must receive a string category even if engine.category=None.
+    # Use canonical group (engine is SoT for group determination)
+    canonical_group = canonical.get("group") or "general"
+    engine_category = canonical.get("category")
+
+    # For pregnant, legacy /plan returns category=None (preserved)
+    # Normalize pregnant flag inline for category=None check only
+    # IMPORTANT: athlete keywords must NEVER imply pregnant=True
+    _YES_VALUES_DEFAULT = {"yes", "y", "true", "1", "да", "д", "si", "sí"}
+    _YES_VALUES_ATHLETE = _YES_VALUES_DEFAULT | {"спортсмен", "athlete"}
```

Flake8 (7.3.0)

[error] local variable '_YES_VALUES_ATHLETE' is assigned to but never used

(F841)